examples/spark/README.md

```diff
@@ -228,9 +255,21 @@ zeppelin-controller-ja09s   1/1       Running   0          53s
 Now you have two choices, depending on your predilections. You can do something
 graphical with the Spark cluster, or you can stay in the CLI.
 
+For both choices, we will be working with this Python snippet:
```

Comment:
The same snippet is repeated below for zeppelin. Perhaps it would be sufficient to say that the same can be run non-interactively also.